docs/_data/sidebar.yml

```diff
@@ -94,7 +94,6 @@
         jp: リファクタリング
       url: /apex/refactoring
     - title:
```

Comment:
If we are going to be removing the Quick Fix section, I think we need to also remove the following:

    - title:
        en: Quick Fix
        jp: クイック修正
      url: /apex/quick-fix

Reply:
From reading https://salesforce.quip.com/HXC1Ap7P0n53, it sounds like the Japanese page might need to stay for longer. Is that correct?

Reply:
@hbc-bindu this is the reason why we get those blank spaces in the sidebar. I'll add them back for now.

Reply:
@lcampos, I wanted to clarify how to resolve the side bar issue. 
Because deleting only the English topics and retaining the Japanese topics is causing those blank lines, should I delete the Japanese topics too and replace the older Japanese topics with updated English topics (for ex, as we have for Deploy on Save)?
cc @smaddox-sf

Reply:
@smaddox-sf , @lcampos 
I deleted Quick Fix and Apex Lang Server topics from the Side bar. And edited the Japanese Apex Writing topic to remove the link to Apex Lang Server. 
Can you please approve this review. Thanks!